apps/client/src/features/rundown/rundown-header/RundownHeader.tsx

```diff
@@ -35,6 +66,7 @@ export default function RundownHeader() {
           aria-label='Edit mode'
         />
       </ButtonGroup>
+      <span style={{ opacity: '30%' }}>{inClipBoard}</span>
```

Comment:
@cpvalente any comments on the look of this?
![image](https://github.com/cpvalente/ontime/assets/19875125/a8123e0b-03ea-4262-b4aa-5ddd2e3bd8f9)

Reply:
hm, I am not sure what I am looking at, but if this is some tip to what i being copied, I dont think it belongs here